vhdbuilder/packer/init-variables.sh

```diff
@@ -109,13 +109,17 @@ if [[ "${MODE}" == "linuxVhdMode" ]]; then
 		if [[ "${OS_SKU}" == "Ubuntu" ]]; then
```

Comment:
nit: can now probably un-nest this to:

` if [[ "${OS_SKU}" == "Ubuntu" && "${IMG_SKU}" == "20_04-lts-cvm" ]]`

Reply:
fair enough, addressed!